docs/guide/middleware.md

```diff
@@ -19,7 +19,7 @@ router: {
 }
 ```
 
-In case of global usage, You can set `auth` option to `false` in a specific component and the middleware will ignore that route.
+In case of global usage, You can set `auth` option to `false` in a specific component and the middleware will ignore that route and will not redirect to any other route. This also applies to `redirect.login` route.
```

Comment:
I propose removing the last sentence, as there is nothing special about the redirect.login route. If you want to explicitly mention the redirect routes, you should say "This also applies to any of the specified `redirect` routes."

Reply:
I rethink a bit. What I want to explicitly mention is "it also applies to the redirect after `loggedIn` state changes". This is what confused me in the first place, Is that ok?